pxr/usd/plugin/usdMtlx/reader.cpp

```diff
@@ -289,7 +290,11 @@ _FindMatchingNodeDef(
         }
 
         // Filter by types.
-        if (mtlxInterface && !mtlxInterface->isTypeCompatible(mtlxNodeDef)) {
+        if (mtlxInterface && !mtlxInterface->hasExactInputMatch(mtlxNodeDef)) {
```

Comment:
This is just a query: This assumes the latest API. I'm curious if code which handles multiple versions is not desirable e.g. if version preprocessor statements ?